### PR TITLE
1.1.0 branch - added property for aspnet template version

### PR DESCRIPTION
**Pull request by seancpeters**
This PR is the rel/1.1.0 analogue of the similar PR for release/15.5 #7811

### CODE REVIEW

**Comment by natemcmaster** [comment]
FYI - I made this change _and_ upgraded the package version to the latest 1.x templates here: https://github.com/dotnet/cli/pull/7837.

File: build/Microsoft.DotNet.Cli.BundledTemplates.props [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <BundledTemplate Include="Microsoft.DotNet.Common.ItemTemplates" Version="$(TemplateEngineTemplateVersion)" />
     <BundledTemplate Include="Microsoft.DotNet.Common.ProjectTemplates.1.x" Version="$(TemplateEngineTemplateVersion)" />
     <BundledTemplate Include="Microsoft.DotNet.Test.ProjectTemplates.1.x" Version="$(TemplateEngineTemplateVersion)" />
-    <BundledTemplate Include="Microsoft.DotNet.Web.ProjectTemplates.1.x" Version="$(TemplateEngineTemplateVersion)" />
+
```
Comment (jzeferino) [inline build/Microsoft.DotNet.Cli.BundledTemplates.props]: This space was unnecessary here.

**Comment by livarcocc** [comment]
It seems this change is no longer necessary, @seancpeters can you confirm?

**Comment by seancpeters** [comment]
This line should be completely removed:
`<BundledTemplate Include="Microsoft.DotNet.Web.ProjectTemplates.1.x" Version="$(TemplateEngineTemplateVersion)" /> `

- because the web templates no longer originate from dotnet/templating.

I don't know about the other change.

**Comment by seancpeters** [comment]
I should clarify: the web templates with the version AspNetTemplateVersion replaces the line I referenced above. The new line is needed.

**Comment by natemcmaster** [comment]
This PR is stale. Not needed anymore.

**Pull request closed without merging**